aws-hybrid-vpn-gcp/inputs.tf

```diff
@@ -0,0 +1,30 @@
+variable "name" {
+  description = "Name of the Hybrid VPN deployment, used in various resources to uniquely identify them"
+  type        = string
+  default     = "test"
```

Comment:
I usually try to avoid any and all default values in the core/internal module. Not saying I never do it but just checking that this is strictly necessary?